main.js

```diff
@@ -4,17 +4,18 @@ const fastify = require('fastify');
 
 const path = require('node:path');
 
-const { Logger } = require('./src/logger.js');
+const { Logger, StreamForLogger } = require('./src/logger.js');
 const http = require('./src/http.js');
 const ws = require('./src/ws.js');
-
 const { loadApplication } = require('./src/loader.js');
+
 const APPLICATION_PATH = path.join(process.cwd(), '../NodeJS-Application');
+const LOG_FOLDER_PATH = './log';
+const streamForLogger = new StreamForLogger(LOG_FOLDER_PATH);
```

Comment:
Move this to line 16.

Reply:
@nechaido do you mean move rows 13, 14 inside IIFE?

Reply:
Done